lib/models/Auction.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose');
+
+const schema = new mongoose.Schema({
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+    required: true
+  },
+
+  title: {
+    type: String,
+    required: true
+  },
+
+  description: {
+    type: String, 
+    required: true
+  },
+  
+  quantity: {
+    type: Number, 
+    required: true
+  },
+
+  endDate: {
+    type: Date, 
+    required: true 
+  }
+// also needs endTime but not sure how to do that 
```

Comment:
endDate is endTime. Its how you want to name the field and you just chose endDate, which is fine.